jenkins-integration/beaker/install/shared/35_customize_hocon_settings.rb

```diff
@@ -0,0 +1,28 @@
+require 'json'
+
+def set_hocon_setting(file_path, setting_path, value)
+  step "Set #{setting_path} in #{file_path}"
+  on(master, "hocon -f #{file_path} set #{setting_path} #{value}")
+end
+
+def unset_hocon_setting(file_path, setting_path)
+  step "Unset #{setting_path} in #{file_path}"
+  on(master, "hocon -f #{file_path} unset #{setting_path}")
+end
+
+test_name 'Configure hocon settings on SUT'
+
+hocon_settings = JSON.parse(ENV['PUPPET_GATLING_HOCON_SETTINGS'])
+
+p hocon_settings
+hocon_settings.each do |setting|
+  case setting["action"]
+    when 'set'
+      set_hocon_setting(setting['file'], setting['path'], setting['value'])
+    when 'unset'
+      unset_hocon_setting(setting['file'], setting['path'])
+    else
+      raise Exception.new("Invalid hocon action '#{setting["action"]}'. Must be one of 'set', 'unset'")
```

Comment:
I think defaulting to "set" is a good idea. I'm not sure if we need to log anything though, since there will be a beaker step in the log saying "Set X in file Y"